core/templates/dev/head/exploration_editor/feedback_tab/ThreadDataService.js

```diff
@@ -69,6 +70,11 @@ oppia.factory('threadDataService', [
         }
       }
     });
+    var payload = {
+      exploration_id: _expId,
+      thread_id: threadId
+    };
+    $http.post(_FEEDBACK_THREAD_VIEW_EVENT_URL, payload);
```

Comment:
Actually, I just realized that this is misplaced (I think). Can it go into the controller function that is triggered when the user actually hits the button to see the new message? I'm concerned about this happening "behind the scenes" and marking stuff as read when it hasn't been.